src/DotLiquid/StandardFilters.cs

```diff
@@ -224,6 +240,18 @@ public static string TruncateWords(string input, int words = 15, string truncate
         /// <returns></returns>
         public static string[] Split(string input, string pattern)
         {
+            // If the pattern is empty and the input is a string convert to an array ...
+            // ... as specified in the Liquid Reverse filter example.
+            if (string.IsNullOrEmpty(pattern) && !string.IsNullOrEmpty(input))
+            {
+                var stringList = new List<string>();

```

Comment:
Less allocations if you create a `char[]` with the correct size, and then use a for loop to iterate the chars.

Reply:
Replaced by code from microalps.